test/unit/matrix/test_util_matrix.cpp

```diff
@@ -162,7 +162,8 @@ void check_is_hermitian(Matrix<const T, Device::CPU>& matrix, comm::Communicator
           return dlaf::conj(tile_original({index.col(), index.row()}));
         };
 
-        CHECK_TILE_NEAR(transposed_conj_tile, sync_wait(keep_future(tile_transposed)).get(),
+        CHECK_TILE_NEAR(transposed_conj_tile,
+                        sync_wait(dlaf::internal::keepFuture(tile_transposed)).get(),
```

Comment:
Not sure about this line, I might be missing something. To me it seems a mixed solution, I would expect

```suggestion
                        tile_transposed.get(),
```

while there is both `sync_wait` treating it as a sender, and `.get()` which makes me think about it as a future.

Reply:
Thanks for the explanation offline!

I was missing the point that `sync_wait` waits on `keep_future`, which is a sender that sends a future. So, once we get the value sent with `sync_wait`, we actually get a future, on which we have to do the `.get()` in order to extract the value from it.